website/developer-docs/docs/writing-documentation.md

```diff
@@ -9,50 +9,61 @@ the documentation is easy to read and uses similar phrasing.
 
 Requirements:
 
--   Node 16 (or greater)
+-   Node.js 16 (or greater)
 
 The documentation site is situated in the `/website` folder of the authentik GitHub repository.
 
 The site is built using npm, below are some useful make commands:
 
--   Install: `make website-install` (Needed for any of the other tasks)
--   Formatting: `make website-lint-fix` or `make website` (Run this before committing)
--   Live editing: `make website-watch` (For real time viewing of changes)
+-   **Installation**: `make website-install`
 
+    This command is rquired before running any of the following commands, and after upgrading any dependencies.
+
+-   **Formatting**: `make website` or `make website-lint-fix`
```

Comment:
while `make website-lint-fix` does run the formatting, `make website` runs the formatting and also a full build, which can surface additional errors that just formatting might not show

Reply:
@BeryJu , should we perhaps not even mention the command `make wesbite-lint-fix` since we really want them to do the full build? Or document both, and explain difference?